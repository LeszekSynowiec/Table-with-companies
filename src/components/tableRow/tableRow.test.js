import React from 'react'
import {shallow} from 'enzyme'
import TableRow from './tableRow.component'

describe('TableRow component', ()=> {
    it('shoud render with out crashing', () => {
        const company = {
            id: 11,
            name: 'testName',
            city: 'testCity',
            incomeTotal: 422,
            incomeAverge: 322,
            incomesLastMonth: 222
        }
        const component = shallow(<TableRow company={company} />)

        expect(component).toMatchSnapshot();
    })
})