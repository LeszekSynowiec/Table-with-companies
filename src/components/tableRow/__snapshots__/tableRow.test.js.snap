// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow component shoud render with out crashing 1`] = `ShallowWrapper {}`;
